### ISSUE TITLE
Add: Antena 3 

### ISSUE THREAD

**Issue by ovidiuux** [iptv-org/iptv#11018]
### Channel ID

Antena3.ro

### Stream URL

https://www.hlsrundle-stream-iptv.gq/api/hls/antena3.m3u8

### Notes

Hosted by me, I will **keep this server alive as long as I can**. I need support to support these servers [I also work on many other TV stations that have token streaming.](https://github.com/ovidiuux/RundleTV-API) Currently due to limited time, severe requirements, **some channels** (only 2) available in a video player program such as VLC Player. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by ovidiuux** [comment]
You can use platform: https://rundle-live.vercel.app/
I'm fix the hls/m3u8 for iptv-org in this days

**Comment by AntiPontifex** [comment]
Dead link.

**Comment by ovidiuux** [comment]
> Dead link. In fact (in Canada), none of the hlsrundle links are working :(

Yeah... i stopped the domain..www.hlsrundle-stream-iptv.gq, 77 gb of bandwidth in 1 days, limit 100 gb.  I turned back again in 1 january 2022. But link are woking. 
https://rundle-live-psi.vercel.app/ or https://rundle-live.vercel.app/ are working to watch. Just replace https://www.hlsrundle-stream-iptv.gq/api/hls/antena3.m3u8 with https://rundle-live.vercel.app/api/hls/antena3.m3u8 or https://rundle-live-psi.vercel.app/api/hls/antena3.m3u8

**Comment by AntiPontifex** [comment]
Thanks for the update: too bad about the upsurge :(

**Comment by ovidiuux** [comment]
> Thanks for the update: too bad about the upsurge :(
https://www.hlsrundle-stream-iptv.gq/api/hls/antena1 are working in VLC
Now, hls domain are working.
Probably certificate takes longer than 24 hours

**Comment by ovidiuux** [comment]
> Thanks for the update: too bad about the upsurge :(

Happy new year! 2 days and 1T bandwidth usage💀

**Comment by AntiPontifex** [comment]
Damn: happy new year to you, too!

**Comment by movies1585** [comment]
Please IPTV Smarter m3u URL send

**Comment by freearhey** [comment]
https://rundle.deta.dev/hls/LIVE$Antena3/6.m3u8/Level(19012010)?start=LIVE&end=END
<img width="791" alt="image" src="https://user-images.githubusercontent.com/7253922/234403646-a227cd29-48b1-43a5-9827-710463aa0d04.png">

**Comment by freearhey** [comment]
https://rundle.deta.dev/hls/LIVE$Antena3/6.m3u8/Level(19012010)?start=LIVE&end=END (Offline)
<img width="617" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/ee42614c-b99d-4ee7-8c8b-5ee1d63a235b">
